### PR TITLE
Feature/sf characteristic

### PR DESCRIPTION
**Pull request by Michal-Dziedzinski**
# Related issue

#164
# Scope of work

Add SfCharacteristic component which contains SfIcon and text part with Title and description. All of that can be changed by using props or slots.
# Screenshots of visual changes
image
# Checklist

- [ ] I followed [composition rules](https://github.com/DivanteLtd/storefront-ui/blob/master/docs/component-rules.md) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/molecules/SfCharacteristic/SfCharacteristic.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
...5 5.76207 0.672852 4.59826 1.84877C3.43445 3.02472 2.78464 4.61405 2.79315 6.26878V9.77605H0.620706C0.277615 9.77605 8.26965e-06 10.0537 8.26965e-06 10.3967V22.8108C-0.00120395 23.0472 0.130938 23.2629 0.341889 23.3696L6.85921 26.5036C8.22062 27.1655 9.81116 27.1655 11.1724 26.5036L17.6581 23.3696C17.8691 23.263 18.0012 23.0472 18 22.8108V10.3967C18 10.2319 17.9345 10.0743 17.8181 9.95788C17.7018 9.8415 17.5442 9.77604 17.3793 9.77604H15.2068V6.26877C15.2153 4.61398 14.5655 3.02467 13.4017 1.84876C12.2379 0.672809 10.6546 0.00847257 8.99974 0L9.00025 1.24309e-05ZM4.03467 6.26878C4.03467 4.49519 4.98148 2.85617 6.51746 1.96888C8.05344 1.08158 9.94712 1.08146 11.483 1.96888C13.019 2.85629 13.9658 4.49531 13.9658 6.26878V9.77605H4.03467V6.26878ZM16.759 22.407L10.6138 25.3869C9.59544 25.8815 8.40494 25.8815 7.38674 25.3869L1.24153 22.4385V11.0173H16.759V22.407Z M9.01975 21.1298C9.22163 21.1298 9.41461 21.0497 9.55711 20.9072C9.69961 20.7647 9.77976 20.5717 9.77976 20.3698V17.3298C9.77976 16.9097 9.43984 16.5698 9.01976 16.5698C8.59969 16.5698 8.25977 16.9097 8.25977 17.3298V20.3698C8.25977 20.5717 8.33992 20.7647 8.48242 20.9072C8.62492 21.0497 8.81789 21.1298 9.01978 21.1298H9.01975Z"
```
Comment (filrak) [inline packages/vue/src/components/molecules/SfCharacteristic/SfCharacteristic.js]: Use `SfIcon` component

File: packages/shared/styles/components/SfCharacteristic.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
+@import '../variables';
+
+$characteristic-margin-right: 20px;
```
Comment (leomp12) [inline packages/shared/styles/components/SfCharacteristic.scss]: must use `!default` for scss vars:

```suggestion
$characteristic-margin-right: 20px !default;
```

File: packages/shared/styles/components/SfCharacteristic.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  align-items: center;
+
+  &__icon {
+    display: flex;
+    align-items: center;
+    margin-right: $characteristic-margin-right;
+  }
+
+  &__title {
+    font-size: $font-size-small-desktop;
```
Comment (leomp12) [inline packages/shared/styles/components/SfCharacteristic.scss]: imo it should be a component variable:

```scss
$characteristic__title-font-size: $font-size-small-desktop !default;
```

```suggestion
    font-size: $characteristic__title-font-size;
```

Comment (leomp12) [reply packages/shared/styles/components/SfCharacteristic.scss]: wdyt?

File: packages/shared/styles/components/SfCharacteristic.scss [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    margin: 0;
+  }
+
+  &__description {
+    font-size: $font-size-extra-small-desktop;
```
Comment (leomp12) [inline packages/shared/styles/components/SfCharacteristic.scss]: also a component level var...

File: packages/vue/src/components/molecules/SfCharacteristic/SfCharacteristic.js (hunk 1 above)
Comment (leomp12) [reply packages/vue/src/components/molecules/SfCharacteristic/SfCharacteristic.js]: and imo the prop should be named `icon` or `iconPath`, I think `path` is not explanatory in this case

File: packages/shared/styles/components/SfCharacteristic.scss (hunk 3 above)
Comment (Michal-Dziedzinski) [reply packages/shared/styles/components/SfCharacteristic.scss]: Sure, I agree with you :)

**Comment by przspa** [comment]
@Michal-Dziedzinski please check css's ;)

**Comment by Michal-Dziedzinski** [comment]
> 
> @Michal-Dziedzinski please check css's ;)
I think that CSS is ok, the problem is: #224

File: packages/vue/src/components/molecules/SfCharacteristic/SfCharacteristic.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<div class="sf-characteristic">
+  <!--@slot Icon. Slot content will replace SfIcon atom component-->
+  <slot name="icon" v-bind="{color, size, iconPath}">
+    <div class="sf-characteristic__icon">
+      <SfIcon :color="color" :size="size" :path="iconPath" />
+    </div>
+  </slot>
+  <!--@slot Characteristic text. Slot content will replace default title and description text-->
+  <slot name="text">
```
Comment (przspa) [inline packages/vue/src/components/molecules/SfCharacteristic/SfCharacteristic.html]:
```suggestion
  <slot name="text" v-bind="{title, description}">
```

**Pull request merged**